openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
@aslakknutsen how about we merge this as it is, we can then do testing of jenkins stuff in our CD pipeline before we release - then I'll look at refactoring to reuse the `controller.InitTenant` later on?